keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/DeliveryArtifact.kt

```diff
@@ -0,0 +1,38 @@
+package com.netflix.spinnaker.keel.api
+
+enum class ArtifactType {
+  deb, docker
```

Comment:
Was changing this to `UPPER` to `lower` to simplify (de-)serialization, or some other reason? Feels like it breaks naming conventions...

Reply:
Exactly that. I'm not totally happy with doing this but I'm not sure if there's another way to avoid having to use `@JsonValue`.

Reply:
I'll see if I can figure another way to do it and revert that part of the change.